source/_integrations/deluge.markdown

```diff
@@ -11,6 +11,8 @@ ha_domain: deluge
 ha_platforms:
   - sensor
   - switch
+
+To be able to use this integration, you have to enable the following option in deluge settings: Daemon > Allow remote controls
```

Comment:
This is not a part of the document.
This information needs to be moved to under the platform listing over `## Sensor`

Reply:
Why I put it there is because this configuration is needed sensors and switch too.
Should duplicate and move it to both places or can I keep it here?

Reply:
<https://deploy-preview-19236--home-assistant-docs.netlify.app/integrations/deluge/> This is the preview, so no you can't keep it here.

If you move it to where I said (above the `## Sensor` entry) it "applies" to both.

Reply:
oh, I see, thanks for your help! :)